eng/testing/tests.singlefile.targets

```diff
@@ -17,6 +17,33 @@
     <PackageReference Include="xunit.runner.utility" Version="2.4.1" />
   </ItemGroup>
 
+  <Target Name="__ExcludeAssembliesFromSingleFile"
+          Inputs="%(ResolvedFileToPublish.Identity)"
+          Outputs="__NewResolvedFiles"
+          BeforeTargets="_ComputeFilesToBundle">
```

Comment:
We usually try to not depend on private targets which are implementation detail and could change their name or be removed. Is there a public target that runs before this or on the chain of this target that you can hook to?

Reply:
Not at the moment, but I think we could probably add something. I'm open to adding a lot more integration points here to help with testing.